src/routes/about/__tests__/get-about.e2e-spec.ts

```diff
@@ -2,6 +2,8 @@ import { INestApplication } from '@nestjs/common';
 import { Test } from '@nestjs/testing';
 import * as request from 'supertest';
 import { AppModule } from '../../../app.module';
+import { expect } from '@jest/globals';
+import '../../../__tests__/matchers/to-be-string-or-null';
```

Comment:
As per our discussion, let's try to move this to the Jest configuration so it need not be imported directly.